scipy/_lib/_uarray/__init__.py

```diff
@@ -33,7 +33,7 @@
 creating multimethods are mostly orthogonal activities, and knowing
 one doesn't necessarily require knowledge of the other, although it
 is certainly helpful. We expect core API designers/specifiers to write the
-multimethods, and implementors to override them. But, as is often the case,
+multimethods, and implementers to override them. But, as is often the case,
```

Comment:
Still not enthusiastic about this. I'd prefer to not warn on rare words by default, but then again, if we're only talking about one legitimate occurrence of `-ors` (the convertor one is even more of an outlier), then it's not all that relevant. 🤷

Reply:
That's OK, I can revert this one. In the event of automation of spell checking later on, such words can be added to an allowlist.

That said, I believe it's beneficial to use spelling that is as standard as possible when writing documentation, both for readers and to avoid alerts from spell checkers such as _aspell_. However, that's just my opinion.